eng/common/TestResources/New-TestResources.ps1

```diff
@@ -329,9 +329,9 @@ try {
         $sp.Id
     }
 
-    # If the ServiceDirectory is an absolute path use the last directory name
-    # (e.g. D:\foo\bar\ -> bar)
-    $serviceName = if (Split-Path -IsAbsolute $ServiceDirectory) {
+    # If the ServiceDirectory has multiple segments use the last directory name
+    # e.g. D:\foo\bar -> bar or foo/bar -> bar
+    $serviceName = if (Split-Path $ServiceDirectory) {
```

Comment:
Not sure if it will matter but go also has some cases where the last part might not be unique.

Reply:
Originally I was going to set it to replace slash characters with underscores, but then I saw this inner comment and figured the convention might instead be to use the leaf path only. I'd be in favor of changing that convention instead to avoid future duplication problems.

Reply:
e.g. for absolute path, use leaf, for multi-segment, use full with replace?

Reply:
I would stick with this for now and we can figure out another strategy if and when we hit a real issue.